### ISSUE TITLE
Error in CSVParser API docs

### ISSUE THREAD

**Issue by mediabeastnz** [silverstripe/silverstripe-framework#5004]
Example code on the CSVParser API docs has a minor error.

Missing array() and comma on the mapColumns() functions.

Should be

```php
$parser->mapColumns(
    array(
        'first name' => 'FirstName',
        'lastname' => 'Surname',
        'last name' => 'Surname'
    )
);
```

**Comment by tractorcow** [comment]
Can you please PR a fix to the docs?

**Comment by kinglozzer** [comment]
Closed via 20d116757a74a87803bc9a933ed5c7b2c3156f04
